index.js

```diff
@@ -4,86 +4,14 @@
 (function () {
   'use strict';
 
-  var async = require('async'),
-    debug = require('debug')('diehard'),
-    assert = require('assert'),
-    handlers = [],
-    diehard = global.diehard;
+  var Diehard = require('./diehard');
 
-  if (!diehard) {
-    module.exports.version = require('./package.json').version;
-    diehard = global.diehard = module.exports;
-  } else {
-    assert(diehard.version === require('./package.json').version, 'Can only load one diehard');
-    module.exports = diehard;
-    return;
+  /*jslint nomen: true*/
+  if (!global._diehardHandlers) {
```

Comment:
If an older version of diehard is around too, we're in a bad place, but since this seems to have been exclusively adopted inside lanetix (for now), we should be OK.